src/gpu/d3d11/SDL_gpu_d3d11.c

```diff
@@ -52,7 +49,6 @@
 #define __stdcall __attribute__((__stdcall__))
 #endif
 #endif
-#define D3DCOMPILER_API __stdcall
```

Comment:
This whole stdcall block can be moved to the single-file header's implementation section.